content/doc/developer/publishing/requesting-hosting.adoc

```diff
@@ -62,12 +63,42 @@ Alternately, you can configure your plugin to be released automatically whenever
 In this case you only need write permission to GitHub; you never need local credentials.
 Follow link:/redirect/continuous-delivery-of-plugins[this guide] for details.
 
-== Categorize the Plugin
+=== Automate your GitHub repository
+
+==== Activate GitHub actions
```

Comment:
change the hosting bot to do this?

Reply:
Can you activate actions via the API? I don't mean in settings, but clicking the green surface "I understand my workflows" 🤔

Reply:
unsure